src/tools/rbd/action/Bench.cc

```diff
@@ -293,18 +336,36 @@ int execute(const po::variables_map &vm) {
     return r;
   }
 
-  r = do_bench_write(image, bench_io_size, bench_io_threads, bench_bytes,
-                     bench_random);
+  r = do_bench(image, bench_io_type, bench_io_size, bench_io_threads,
+		     bench_bytes, bench_random);
   if (r < 0) {
-    std::cerr << "bench-write failed: " << cpp_strerror(r) << std::endl;
+    std::cerr << "do_bench failed: " << cpp_strerror(r) << std::endl;
     return r;
   }
   return 0;
 }
 
-Shell::Action action(
-  {"bench-write"}, {}, "Simple write benchmark.", "", &get_arguments, &execute);
+int execute_for_write(const po::variables_map &vm) {
+  return bench_execute(vm, bench_write);
+}
+
+int execute_for_bench(const po::variables_map &vm) {
+  enum io_type bench_io_type;
+  if (vm.count("io-type")) {
+    bench_io_type = vm["io-type"].as<enum io_type>();
+  } else {
+    bench_io_type = bench_read;
+  }
+  return bench_execute(vm, bench_io_type);
+}
+
+Shell::Action action_write(
+  {"bench-write"}, {}, "Simple write benchmark. (Deprecated, please use `rbd bench IMAGE --io-type write` instead.)",
```

Comment:
May be just "please use `rbd bench` instead"  to make it shorter? I think this info will be enough for the user to figure out the right command.